backend/src/packages/auth/auth.controller.ts

```diff
@@ -30,7 +30,7 @@ class AuthController extends Controller {
     this.authService = authService;
 
     this.addRoute({
-      path: AuthApiPath.SIGN_UP_CUSTOMER,
+      path: AuthApiPath.SIGN_UP,
       method: 'POST',
       validation: {
```

Comment:
Doesn't it ruin a sign-up logic?

Reply:
removed it, it was mock from when signup didn't work in another way